assets/js/blocks/classic-template/test/utils.ts

```diff
@@ -41,6 +23,24 @@ describe( 'getTemplateDetailsBySlug', function () {
 		).toStrictEqual( TEMPLATES[ 'single-product' ] );
 	} );
 
+	it( 'should return taxonomy-product object when given a partial match', () => {
+		expect(
+			getTemplateDetailsBySlug( 'taxonomy-product_tag', TEMPLATES )
+		).toBeTruthy();
+		expect(
+			getTemplateDetailsBySlug( 'taxonomy-product_tag', TEMPLATES )
+		).toStrictEqual( TEMPLATES[ 'taxonomy-product_tag' ] );
```

Comment:
Nitpicking, but doesn't the second assertion cover the first one? I mean, if the output should strictly equal a specific value, then it has to be truthy as well.

I don't know your opinion, but I think more straightforward tests make it easier to maintain and understand the expected outcome. If you agree, then:

```
		expect(
			getTemplateDetailsBySlug(<<NAME>>, TEMPLATES )
		).toBeTruthy();
```

part could be removed from each of these tests.

Reply:
I just copied and pasted what there was before TBH :D. I agree with you! I'm going to remove the first check.